examples/cpp/load_balancing/README.md

```diff
@@ -0,0 +1,64 @@
+# gRPC C++ Load Balancing Tutorial
+
+### Prerequisite
+Make sure you have run the [hello world example](../helloworld) or understood the basics of gRPC. We will not dive into the details that have been discussed in the hello world example.
+
+### Get the tutorial source code
+
+The example code for this and our other examples lives in the `examples` directory. Clone this repository to your local machine by running the following command:
+
+
+```sh
+$ git clone -b $(curl -L https://grpc.io/release) https://github.com/grpc/grpc
+```
+
+Change your current directory to examples/cpp/load_balancing
+
+```sh
+$ cd examples/cpp/load_balancing/
+```
+
+### Generating gRPC code
+
+To generate the client and server side interfaces:
+
+```sh
+$ make helloworld.grpc.pb.cc helloworld.pb.cc
+```
+Which internally invokes the proto-compiler as:
+
+```sh
+$ protoc -I ../../protos/ --grpc_out=. --plugin=protoc-gen-grpc=grpc_cpp_plugin ../../protos/helloworld.proto
+$ protoc -I ../../protos/ --cpp_out=. ../../protos/helloworld.proto
+```
+
+### Writing a client and a server
+
+The client and the server can be based on the hello world example.
+
+Additionally, we can configure the load balancing policy.
+
+In the client, set the load balancing policy of the channel via the channel arg (to, for example, Round Robin).
+
+```cpp
+  ChannelArguments args;
+  // Set the load balancing policy for the channel.
+  args.SetLoadBalancingPolicyName("round_robin");
```

Comment:
It might be nice to point to where the list of available lb policies are, but I suppose the user could just look at the code

Reply:
Done.